core/src/main/java/com/amplifyframework/core/model/ModelSchema.java

```diff
@@ -113,14 +114,20 @@ public static ModelSchema fromModelClass(@NonNull Class<? extends Model> clazz)
             final TreeMap<String, ModelField> fields = new TreeMap<>();
             final TreeMap<String, ModelAssociation> associations = new TreeMap<>();
             final TreeMap<String, ModelIndex> indexes = new TreeMap<>();
+            final List<AuthRule> authRules = new ArrayList<>();
 
             // Set the model name and plural name (null if not provided)
             ModelConfig modelConfig = clazz.getAnnotation(ModelConfig.class);
             final String modelName = clazz.getSimpleName();
             final String modelPluralName = modelConfig != null && !modelConfig.pluralName().isEmpty()
                     ? modelConfig.pluralName()
                     : null;
-            final boolean hasOwnerAuthorization = modelConfig != null && modelConfig.hasOwnerAuthorization();
+
+            if (modelConfig != null) {
+                for (com.amplifyframework.core.model.annotations.AuthRule ruleAnnotation : modelConfig.authRules()) {
```

Comment:
Remove the import qualification

Reply:
Hmm, I actually can't remove it because there are references to the `AuthRule` class and the `@AuthRule` annotation within ModelSchema.java, so this differentiates which one I am referring to.   I suppose the qualifications could be eliminated with a larger refactor of some business logic, but I don't see a strong reason to do that at this point.

Of note, there are also fully qualified class names for the `@ModelField` annotation within this ModelSchema.java file as well for the same reason (there's a `@ModelField` annotation, and a `@ModelField` class)